src/Microsoft.DotNet.Archive/LZMA/ICoder.cs

```diff
@@ -0,0 +1,157 @@
+// ICoder.h
```

Comment:
This code is a direct copy of the LZMA SDK which is in the public domain.  The source in LZMA didn't have a copyright header.  I don't think its correct to add our header to these.  Are you aware of any precedent?  /cc @joshfree